schema.graphql

```diff
@@ -20,4 +20,10 @@ type Product @table @export {
 type Traits @table @export {
   id: ID @primaryKey
   traits: [String]
+}
+
+type PersonalizeCache @table @export {
```

Comment:
This originally had an expiration date on it to create a caching table, but it behaves much differently than a standard table. I needed to get something working (even if it isn't a true "cache")